### ISSUE TITLE
Error: Update store loop exited

### ISSUE THREAD

**Issue by curquiza** [meilisearch/MeiliSearch#1455]
Version RC1

After 10min of non-activity, I get this error.

```bash
cargo run --release -- --master-key=masterKey
    Finished release [optimized + debuginfo] target(s) in 1.19s
     Running `target/release/meilisearch --master-key=masterKey`

888b     d888          d8b 888 d8b  .d8888b.                                    888
8888b   d8888          Y8P 888 Y8P d88P  Y88b                                   888
88888b.d88888              888     Y88b.                                        888
888Y88888P888  .d88b.  888 888 888  "Y888b.    .d88b.   8888b.  888d888 .d8888b 88888b.
888 Y888P 888 d8P  Y8b 888 888 888     "Y88b. d8P  Y8b     "88b 888P"  d88P"    888 "88b
888  Y8P  888 88888888 888 888 888       "888 88888888 .d888888 888    888      888  888
888   "   888 Y8b.     888 888 888 Y88b  d88P Y8b.     888  888 888    Y88b.    888  888
888       888  "Y8888  888 888 888  "Y8888P"   "Y8888  "Y888888 888     "Y8888P 888  888

Database path:		"./data.ms"
Server listening on:	"http://127.0.0.1:7700"
Environment:		"development"
Commit SHA:		"5c9401ad9441822eddef1cf29d8a46157dc97410"
Commit date:		"2021-06-30"
Package version:	"0.21.0"
Anonymous telemetry:	"Disabled"

A Master Key has been set. Requests to MeiliSearch won't be authorized unless you provide an authentication key.

Documentation:		https://docs.meilisearch.com
Source code:		https://github.com/meilisearch/meilisearch
Contact:		https://docs.meilisearch.com/resources/contact.html or bonjour@meilisearch.com

[2021-07-01T11:56:06Z INFO  actix_server::builder] Starting 12 workers
[2021-07-01T11:56:06Z INFO  actix_server::builder] Starting "actix-web-service-127.0.0.1:7700" service on 127.0.0.1:7700
[2021-07-01T11:56:17Z INFO  actix_web::middleware::logger] 127.0.0.1:53272 "POST /indexes/indexUID/search HTTP/1.1" 200 918 "-" "PostmanRuntime/7.28.1" 0.003051
[2021-07-01T12:06:06Z ERROR meilisearch_http::index_controller::update_actor::store] Update store loop exited.
```

**Comment by curquiza** [comment]
Related to https://github.com/meilisearch/MeiliSearch/issues/1453

**Comment by curquiza** [comment]
Fixed #1456
